Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6-stretch
+FROM node:6.14.4-stretch
```

Comment:
Why is the node version fixed here? Can't we just use `6` or `6.15` to get the security updates?

Reply:
I usually tend to fix versions to avoid that the "generic" image `6` is downloaed one time and never updated (as long nobody performs a `docker pull` on it). In that case the version `6` could even be older as the pinned version.
What do you think?

Reply:
I understand the motivation and I'm not strongly opposed to fixing the version, but I think we could fix just the major version and let the minor version to update (`6.15`) or at least update to the latest version (`6.15.1`) ?

Reply:
Alright. Updated the version to the latest one.